core/src/main/java/bio/terra/pearl/core/service/survey/SurveyTaskDispatcher.java

```diff
@@ -49,10 +50,14 @@ public void createSurveyTasks(EnrolleeEvent enrolleeEvent) {
                 enrolleeEvent.getPortalParticipantUser(),
                 enrolleeEvent.getEnrolleeRuleData(),
                 studyEnvSurveys);
+        DataAuditInfo auditInfo = DataAuditInfo.builder()
+                .systemProcess(getClass().getSimpleName() + ".createSurveyTasks")
```

Comment:
Maybe make a helper for formatting the systemProcess, since it will be pretty common?